integration_tests.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import optuna
+import time
+import threading
```

Comment:
A nit, but how about stick to the alphabetical order?
```suggestion
import threading
import time
```

Reply:
Please ignore my comment if the alphabetical order is not relevant to this project.

Reply:
Thank you for your suggestion!
I fix the order of import statements at b6773e7 👍